src/components/MedicalItemMasterPanel.js

```diff
@@ -106,6 +106,17 @@ class MedicalItemMasterPanel extends FormPanel {
               onChange={(quantity) => this.updateAttributes({ quantity })}
             />
           </Grid>
+          <Grid item xs={2} className={classes.item}>
+            <NumberInput
+              min={0}
+              module="admin"
+              label="medical.item.maximumAmount"
+              name="maximumAmount"
+              readOnly={readOnly}
+              value={edited && edited.maximumAmount ? edited.maximumAmount : ""}
```

Comment:
Using optional chaining here will make this value line shorter:
`value={edited?.maximumAmount ?? ""}`